### PR TITLE
Added link to register from login page

### PR DESCRIPTION
**Pull request by ganeshpatro321**
@root5533 Please Review

### CODE REVIEW

**Comment by root5533** [comment]
Center the text. Reduce the font size (around 9px will be alright but it's up to you). Set the hyperlink only on the sign up and not the whole sentence. Also have a margin between the text and button.

**Comment by ganeshpatro321** [comment]
We will have a forgot password button also, centering the text will create problem maybe? @root5533

**Comment by ganeshpatro321** [comment]
screenshot from 2019-03-08 22-56-29

It looks like this now.
Will I centre the text?
@root5533

File: fact-bounty-client/src/pages/login/components/Login.js [hunk 1]
```diff
@@ -139,6 +142,9 @@ class Login extends Component {
 								color="primary" className={this.props.classes.submit}>
 							Login
 						</Button>
+						<div style={{'paddingTop': "6px", "float": "right"}}>
```
Comment (root5533) [inline fact-bounty-client/src/pages/login/components/Login.js]: Don't use inline css. Add a class in `styles` and set className here

File: fact-bounty-client/src/pages/login/components/Login.js [hunk 2]
```diff
@@ -139,6 +142,9 @@ class Login extends Component {
...
+						Don't have an account? <Link component={RouterLink} to="/register" style={{'color': 'red'}}>Sign up!</Link>
```
Comment (root5533) [inline fact-bounty-client/src/pages/login/components/Login.js]: Same here.

**Comment by root5533** [comment]
Let's center it and add more space between the button and the register link.

**Comment by ganeshpatro321** [comment]
Okay, I will update with the required changes @root5533

**Comment by ganeshpatro321** [comment]
@root5533 Please review

**Pull request merged**